src/Facilitator/index.js

```diff
@@ -795,6 +815,12 @@ class Facilitator {
       );
       return Promise.reject(err);
     }
+    const latestAnchorInfo = await this.gateway.getLatestAnchorInfo();
+    if ((new BN(latestAnchorInfo.blockHeight)).lt(new BN(blockNumber))) {
+      logger.error('Latest available state root on target is lower than the height at which redeem was done!');
+      const err = new TypeError(`Latest available state root on target is lower than the height at which redeem was done!`);
```

Comment:
Similarly, this should be Error instead of TypeError.
Also, update the error message

Reply:
Done!